.github/workflows/budibase_ci.yml

```diff
@@ -151,7 +153,19 @@ jobs:
         with:
           node-version: 20.x
           cache: yarn
+
+      - name: Pull testcontainers images
+        run: |
+          docker pull mcr.microsoft.com/mssql/server:2022-latest
+          docker pull mysql:8.3
```

Comment:
The mysql does not align with the actual usage. I see that we have plenty of `GenericContainer` usages without the image tag. The same happens with mongo, pg, redis. We should align them, otherwise we will be downloading multiple images on test tile